robot/client/client_test.go

```diff
@@ -512,7 +506,7 @@ func TestStatusClient(t *testing.T) {
 
 	camera1, err := camera.FromRobot(client, "camera1")
 	test.That(t, err, test.ShouldBeNil)
-	_, _, err = camera.ReadImage(context.Background(), camera1)
+	_, _, err = camera1.Image(context.Background(), rutils.MimeTypeJPEG, nil)
```

Comment:
shouldn't we test that this DOESN'T return JPEG bytes and still doesn't error?

Reply:
I added a check to make sure that the bytes and metadata are nil, and empty struct respectively